cucumber.js

```diff
@@ -1,19 +1,26 @@
 require('dotenv').config({
-  path: process.env.NODE_ENV === 'test' ? '.env.test' : '.env'
+  path: process.env.NODE_ENV === 'test' ? '.env.test' : '.env',
 });
 
 const worldParameters = {
   appUrl: process.env.BASE_APP_URL || 'http://localhost:4321',
-  generationServiceUrl: process.env.GENERATION_SERVICE_URL || 'http://localhost:4321',
   authorizationHeader: process.env.AUTH_HEADER || null,
   validatorServiceUrl: process.env.VALIDATOR_SERVICE_URL || null,
-  mockPort: process.env.MOCK_PORT || null
+  generationServiceUrl:
+    process.env.GENERATION_SERVICE_URL || 'http://localhost:4321',
+  repositoryServiceUrl:
+    process.env.RESPOSITORY_SERVICE_URL || 'http://localhost:4321',
+  registryServiceUrl:
+    process.env.REGISTRY_SERVICE_URL || 'http://localhost:4321',
```

Comment:
Do we need this? I wasn't sure if we would need URLs for these services in the future

Reply:
I think they're good to have, since they would be different services in reality. It might be nice to add these to the table in the README so that a user will know that they can be changed.